src/Components/AddFoodForm.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+function AddFoodForm(props) {
+
+    const [name, setName] = useState('');
+    const [image, setImage] = useState('');
+    const [calories, setCalories] = useState('');
+    const [servings, setServings] = useState('');
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+        const newFood = {
+            name: name,
+            image: image,
+            calories: calories,
+            servings: servings,
+        };
```

Comment:
You could have used the syntax below too (as properties and variables have the same name): 
```
 const newFood = {
            name,
            image,
            calories,
            servings,
        };
```